src/main/webapp/js/common/react/tests/checkTesting.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Checks if the code is currently being run in a NodeJS environment, in which
+ * case the code is being tested and the React modules should be imported.
```

Comment:
Let's document some of our offline conversation here. I.e. why this is needed: these imports don't work in the browser. And possible solutions: bundling (Webpack, Browserify).

Reply:
I think you added a little description in the PR, but still didn't explain fully why we need these imports (just a simple "imports don't work in the browser" might have been enough). Both the comments in this file and the PR do a great job explaining *what* the code is doing. Though I can also figure that out by reading 4 lines of code :)

Reply:
Good point. The _what_ versus the _why_. I added a bit more context for the why and some of the alternatives considered.